MAINTAINERS.yml

```diff
@@ -0,0 +1,1166 @@
+         - tests/lib/c_lib/
+    labels:
+         - "area: C Library"
+
+CMSIS API layer:
+    status: orphaned
+    collaborators:
+         - nashif
+    files:
+         - lib/cmsis_rtos_v*/
+
+Common arch:
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - arch/common/
+        - include/arch/common/
+    labels:
+        - "area: Architectures"
+
+Debug:
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - include/debug/
+        - subsys/debug/
+    labels:
+        - "area: Debugging"
+
+DFU:
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - include/dfu/
+        - subsys/dfu/
+        - tests/subsys/dfu/
+
+Devicetree:
+    status: maintained
+    maintainers:
+        - galak
+    collaborators:
+        - mbolivar-nordic
+    files:
+        - scripts/dts/
+        - dts/bindings/
+        - dts/common/
+    labels:
+        - "area: Device Tree"
+
+Disk:
+    status: orphaned
+    files:
+        - subsys/disk/
+        - include/disk/
+    labels:
+        - "area: Disk Access"
+
+Display drivers:
+    status: maintained
+    maintainers:
+        - vanwinkeljan
+    collaborators:
+        - jfischer-phytec-iot
+    files:
+        - drivers/display/
+        - dts/bindings/display/
+        - include/drivers/display.h
+        - include/display/
+        - include/drivers/display.h
+        - lib/gui/
+        - subsys/fb/
+    labels:
+        - "area: Display"
+
+Documentation:
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - carlescufi
+    collaborators:
+        - nashif
+    files:
+        - doc/
+    files-regex:
+        - \.rst$
+        - /doc/
+    labels:
+        - "area: Documentation"
+
+"Drivers: ADC":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - anangl
+    files:
+        - drivers/adc/
+        - include/drivers/adc.h
+        - tests/drivers/adc/
+    labels:
+        - "area: ADC"
+
+"Drivers: Audio":
+    status: maintained
+    maintainers:
+        - Vudentz
+    files:
+        - drivers/audio/
+        - include/audio/
+
+"Drivers: CAN":
+    status: maintained
+    maintainers:
+        - alexanderwachter
+    collaborators:
+        - henrikbrixandersen
+        - karstenkoenig
+    files:
+        - drivers/can/
+        - dts/bindings/can/
+        - include/drivers/can.h
+        - samples/drivers/CAN/
+        - tests/drivers/can/
+    labels:
+        - "area: CAN"
+
+"Drivers: Clock control":
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers: nordic-krch
+    files:
+        - drivers/clock_control/
+        - dts/bindings/clock/
+        - include/drivers/clock_control.h
+        - include/dt-bindings/clock/
+        - tests/drivers/clock_control/
+    labels:
+        - "area: Clock control"
+
+"Drivers: Console":
+    status: orphaned
```

Comment:
This is not correct entry. There should be 2 entries: "Subsys: Console", with include/console/ and subsys/console/, and myself as a maintainer. And "Drivers: Console" with include/drivers/console/ and drivers/console/. That's part which have wonders like handling line editing in ISR. Glad to hear it's orphaned, should be gone altogether. Feel free to keep me in "collaborators", so somebody watched what happens there.

Reply:
Sure @pfalcon i'll add this entries as you propose (will need to go via TSC approval)